Sources/Basics/Environment/Environment.swift

```diff
@@ -84,6 +84,10 @@ extension [String: String] {
 // MARK: - Path Modification
 
 extension Environment {
+    var path: String? {
```

Comment:
can you use `environment[.path]` instead? its more clear imo that this is a dictionary-ish object

Reply:
Where would `.path` come from?

Reply:
Yeah its a static member of `EnvironmentKey`, for example in UserToolchain:
```swift
            // Get the search paths from PATH.
            self.envSearchPaths = getEnvSearchPaths(
                pathString: environment[.path],
                currentWorkingDirectory: fileSystem.currentWorkingDirectory
            )
```

Reply:
Found it, thanks!